EmcraftSystems/tinyclunx33_zephyr_example
# TNV-36: Add CDC ACM port handler which writes received strings back w…

…ith Zephyr prefix

This PR adds one more CDC ACM UART port.
There's handler running on this port, which receives strings and prints them back with Zephyr prefix.
The string ends with '\r' character or is truncated to 64 bytes

Unit test:
- Open terminal on /dev/ttyACM1 device
- Type 'Test'
- Press Enter
- Observe text 'ZephyrTest` is printed back
